### PR TITLE
Add all spoiled RTTFA player cards

### PR DESCRIPTION
**Pull request by zzorba**
Add new OYO permanent from RTTFA

### CODE REVIEW

File: pack/return/rttfa.json [hunk 1]
```diff
@@ -16,6 +33,134 @@
...
+		"faction_code": "seeker",
+		"illustrator": "Greg Bobrowski",
+		"name": "Ancient Stone",
+		"pack_code": "rttfa",
+		"position": 4,
+		"quantity": 2,
+		"skill_agility": 2,
+		"slot": "Hand",
+		"subname": "Transient Thoughts",
+		"text": "You can only include this asset in your deck by upgrading it from Ancient Stone <i>(Unidentified)</i>, and only if you have \"identified the stone\" in your Campaign Log.\nUses (X secrets). X is the number in parentheses next to \"you have identified the stone.\"\n[reaction] When you draw any number of cards, spend that many secrets: Move that many times.",
```
Comment (robert-huang) [inline pack/return/rttfa.json:66]: \"you have identified the stone\"

looks like it's written wrong for the other ancient stones as well, i'll fix those

File: pack/return/rttfa.json [hunk 2]
```diff
@@ -16,6 +33,134 @@
...
+		"slot": "Hand",
+		"skill_agility": 1,
+		"skill_combat": 1,
+		"text": "Uses (5 ammo).\n[action] Spend 1 ammo: <b>Fight</b>. You get +2 [combat] for this attack. This attack deals +1 damage.\n<b>Forced</b> - At the end of the round, if you triggered Colt Vest Pocket's \"[action]\" ability: Discard it.",
+		"traits": "Item. Weapon. Firearm. Illicit.",
+		"type_code": "asset",
+		"xp": 2
+	},
+	{
+	  "code": "53007",
```
Comment (robert-huang) [inline pack/return/rttfa.json:109]: minor nit, but can you tab this section so it matches the rest of the json?

File: pack/return/rttfa.json (hunk 1 above)
Comment (zzorba) [reply pack/return/rttfa.json:66]: Yep!

File: pack/return/rttfa.json (hunk 2 above)
Comment (zzorba) [reply pack/return/rttfa.json:109]: Will do!

**Pull request merged**